website/content/en/status/report-2023-07-2023-09/valgrind.adoc

```diff
@@ -19,5 +19,6 @@ As usual there are numerous small bugfixes.
 
 Specific to FreeBSD there is now support for FreeBSD 15.
 Two extra `_umtx_op` operations are now supported, `UMTX_OP_GET_MIN_TIMEOUT` and `UMTX_OP_SET_MIN_TIMEOUT`.
-There is a fix for the use of sysctl kern proc pathname with the guest pid or -1 which previously returned the path of the Valgrind host. The sysctl will now return the name of the guest.
+There is a fix for the use of sysctl kern proc pathname with the guest pid or -1, which previously returned the path of the Valgrind host.
```

Comment:
I think, this sentence needs a comma. Is the suggested comma suitably placed?

Reply:
OK for the comma but I think it should still make it clear that the sysctl now returns the path to the guest executable rather than that of the Valgrind tool that it is executing within.

Reply:
Thanks. 

The only other change to this paragraph was (AsciiDoc) markup, one sentence per line. Preview: 

- <https://github.com/grahamperrin/freebsd-doc/blob/patch-2/website/content/en/status/report-2023-07-2023-09/valgrind.adoc>

> … the sysctl now returns the path to the guest executable …

_Name_ or _path_? 

Originally: 

>> … The sysctl will now return the name of the guest. …

Reply:
Let's use path to be consistent.

Reply:
> Let's use path to be consistent.

Resolved by [`b061192` (#260)](https://github.com/freebsd/freebsd-doc/pull/260/commits/b0611924472e4d682c8ea122da2e1d7cef8e4ff2).